worker/Makefile

```diff
@@ -209,17 +207,8 @@ libmediasoup-worker: setup flatc
 	$(MESON) install -C $(BUILD_DIR) --no-rebuild --tags libmediasoup-worker
 
 flatc: setup
-	$(MESON) compile -C $(BUILD_DIR) flatc
-	[ -d $(FLATC_OUT_DIR) ] || \
-		$(FLATC) --cpp --cpp-field-case-style lower --reflect-names --scoped-enums \
-			-o $(FLATC_OUT_DIR) ./fbs/*
-
-flatc-clean:
```

Comment:
No need for this anymore?

Reply:
nop, we don't need a specific clear for flatc anymore. the cleanup is done for the regular 'clean'  target as the generated sources reside in the same build directory.